airflow/contrib/operators/gcs_to_s3.py

```diff
@@ -99,8 +99,12 @@ def execute(self, context):
             # if we are not replacing -> list all files in the S3 bucket
```

Comment:
The "repkace" parameter is not described in the docstring. I think it's a good opportunity to add the docstring